src/adapter/keys.m

```diff
@@ -63,5 +64,5 @@ bool allMandatoryPayloadKeysSet(NSDictionary *data) {
 }
 
 NSArray<NSString *> *identifyingPayloadKeys(void) {
-    return @[ kMRABundleIdentifier, kMRATitle, kMRAArtist, kMRAAlbum ];
+    return @[ kMRABundleIdentifier, kMRAParentAppBundleIdentifier, kMRATitle, kMRAArtist, kMRAAlbum ];
```

Comment:
This doesn't work with the current `isSameItemIdentity` function, streaming will never diff when the parent identifier is nil.